op-chain-ops/contracts/common.go

```diff
@@ -0,0 +1,28 @@
+package contracts
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	// EIP1976ImplementationSlot
```

Comment:
I think this may be duplicated in the go codebases

Reply:
Good catch - duplicated here
https://github.com/ethereum-optimism/optimism/blob/develop/op-chain-ops/genesis/helpers.go#L25-L28